tests/lib/snaps/test-snapd-ssh-public-keys/bin/ssh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/bin/ssh "$@"
```

Comment:
I kind of prefer having snaps that just expose the shell and let you start ssh without an intermediate script. Sometimes those scripts just pass on and call the real target but if logic creeps in they are more complex to reason about than just a simple "run this shell command" from the test.